bika/lims/browser/bika_listing.py

```diff
@@ -239,14 +238,14 @@ def submitTransition(self, action, came_from, items):
                     else:
                         success, message = doActionFor(item, action)
                     if success:
-                        transitioned.append(item.id)
+                        transitioned.append(item.UID())
                     else:
                         self.context.plone_utils.addPortalMessage(message, 'error')
         # automatic label printing
         if transitioned and action == 'receive' \
             and 'receive' in self.portal.bika_setup.getAutoPrintStickers():
             q = "/sticker?template=%s&items=" % (self.portal.bika_setup.getAutoStickerTemplate())
-            # selected_items is a list of UIDs (stickers for AR_add use IDs)
+            # selected_items is a list of UIDs
```

Comment:
Reminder (no action required): curious to see that although secondary-ARs are not printed during register transition, the same principle doesn't apply when the AR is received.